src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1032,7 +1035,7 @@ void M()
 
         #region other parentheses
 
-        private readonly string s_otherParenthesesIgnore = $@"
+        private readonly string s_otherParenthesesAlwaysForClarity = $@"
```

Comment:
note: it feels liek this would benefit from some sort of text in the preview effectively saying: "this will never add parens.  it will just leave all 'other' parens alone".

Reply:
I didn't want to constrain future cases where we decide parentheses would be beneficial for clarity. Severity `none` is the only true way to get "ignore" behavior.

Reply:
With that said, I'm not confident the current text is the best we can make it...